src/components/Breadcrumb/index.jsx

```diff
@@ -1,80 +1,93 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 import cx from 'classnames'
 import styles from './style.styl'
+import arrowLeftIcon from 'assets/icons/icon-arrow-left.svg'
+import palette from 'cozy-ui/stylus/settings/palette.json'
+import { Icon } from 'cozy-ui/react'
 
-const DEFAULT_SEPARATOR = '/'
-const DEFAULT_LAST_SEPARATOR = false
-const DEFAULT_TAG = 'span'
-
-const BreadcrumbSeparator = ({ separator }) => (
-  <span className={styles.Breadcrumb__separator}>{separator}</span>
+const BreadcrumbSeparator = () => (
+  <span className={styles.BreadcrumbSeparator}>/</span>
 )
 
-const BreadcrumbItem = ({ item }) => {
-  const Tag = item.tag
+const BreadcrumbItem = ({
+  name,
+  onClick,
+  isCurrent = false,
+  tag = 'span',
+  showSeparator = false
+}) => {
+  const Tag = tag
   return (
     <div
-      onClick={item.onClick}
-      className={cx(
-        styles.Breadcrumb__crumb,
-        item.isLast && styles.Breadcrumb__last,
-        { [styles.Breadcrumb__link]: item.onClick }
-      )}
+      className={cx(styles.BreadcrumbItem, {
+        [styles['BreadcrumbItem--current']]: isCurrent
+      })}
     >
-      <Tag className={styles.Breadcrumb__title}>
-        {item.name}
-        {item.displaySeparator && (
-          <BreadcrumbSeparator separator={item.separator} />
-        )}
+      <Tag
+        onClick={onClick}
```

Comment:
Curious : why did you move the onClick handler ?

Reply:
Because if I put the `onClick` on the parent, it would also be on the separator. Which seems weird to me.